pendingTranslations.ftl

```diff
@@ -22,3 +22,77 @@ toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
+
+# Icon alts / aria-labels
+chevron-down-alt = Drop down button
+open-in-new-tab-aria-label = Open link in a new tab
+open-in-new-tab-alt = Open button
+
+# Status Pill
+
+status-pill-action-needed = Action Needed
+status-pill-progress = In Progress
+status-pill-fixed = Fixed
+
+# Exposure Card
+
+exposure-card-family-members = Family members
+exposure-card-email = Email
+exposure-card-phone-number = Phone number
+exposure-card-address = Address
+# Variables:
+# $exposure_num is the number of exposures found for a particular type, e.g. 3 found 
+exposure-card-num-found = 
+  { $exposure_num ->
+     *[other] { $exposure_num } found
+  }
+exposure-card-description-info-for-sale-part-one = This site is selling and publishing <data_broker_link>details about you</data_broker_link>
+exposure-card-description-info-for-sale-part-two = Remove this profile to protect your privacy.
+# Variables:
+# $data_breach_company is the company assosciated with the data breach.
+# $data_breach_date is the date of the data breach.
+exposure-card-description-data-breach-part-one = Your information was exposed in the <data_breach_link>{ $data_breach_company } data breach on { $data_breach_date }</data_breach_link>
+exposure-card-description-data-breach-part-two = We'll walk you through the steps to fix it.
+exposure-card-your-exposed-info = Your exposed info
+exposure-card-exposure-type-data-broker = Info for sale
+exposure-card-exposure-type-data-breach = Data breach
+
+# Dashboard
+
+dashboard-top-banner-protect-your-data-title = Let's protect your data
+# Variables:
+# $data_breach_total_num is the total number of data breaches the user has.
+# $data_broker_total_num is the total number of data brokers selling the user's data.
+dashboard-top-banner-protect-your-data-description = We found your data in { $data_breach_total_num } data breaches and { $data_broker_total_num } sites selling your personal info. We'll guide you on how to fix it.
```

Comment:
This string works for English, but it's going to be impossible to localize in many languages (there is no support for multiple plurals). It's also broken in English if any of these is singular.

If possible, I would suggest content to find an alternative solution now, instead of trying to come up with something later on.

Reply:
Correct. As far as I can tell, there isn't any localization framework that supports that.

If you think about it, things get quickly out of hand with 6 possible plurals for each variable (I forgot everything about statistics, but hopefully the math is right):
* 1 plural -> 6 variants
* 2 plurals -> 36 variants

While English only has `one` and `other`, it gets much more complicated for Slavic languages.
https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html

Reply:
For the sake of completeness, technically Fluent supports multiple plurals in one string, but that's deprecated (Message Format 2 doesn't support them, and we want each plural variant to be a fully formed sentence).

This is pretty much impossible for a non-technical person to translate, and you'd have 6 variants instead of 2 for each plural.

```
dashboard-top-banner-protect-your-data-description = 
    We found your data in 
    { $data_breach_total_num ->
        [one] { $data_breach_total_num } data breach
       *[other] { $data_breach_total_num } data breaches
    } 
    and 
    { $data_broker_total_num ->
        [one] { $data_broker_total_num } site
       *[other] { $data_broker_total_num } sites
    } 
    selling your personal info. We’ll guide you on how to fix it.
```

Reply:
It's not ideal (you'd need to "glue" them together with spaces), but it would at least work.